models/articles-model.js

```diff
@@ -36,7 +36,19 @@ exports.selectArticles = () => {
 };
 
 exports.updateArticleVotes = (article_id, inc_votes) => {
+
   const queryString = `UPDATE articles SET votes = votes + $1 WHERE article_id = $2 RETURNING *`;
+  if (!inc_votes) {
```

Comment:
It is highly likely that psql will give you an error code so that you don't need to do custom rejects here.